src/panels/config/integrations/integration-panels/zwave_js/zwave_js-add-node.ts

```diff
@@ -13,6 +13,7 @@ export class DialogZWaveJSAddNode extends HTMLElement {
   connectedCallback() {
     showZWaveJSAddNodeDialog(this, {
```

Comment:
@bramkragten do you know when this is called?

Reply:
It is only used for the `/config/zwave_js/add` route. The intention seems to be to open the add dialog and redirect to `/config/devices/dashboard` but the dialog doesn't open. So it's broken?
Maybe some of the dialog refactoring broke it but if no one has noticed, was it even used?

Reply:
It is probably for the my link?

Reply:
We may have to fix it then. Pretty sure the dialog changes broke it because it does `showDialog` and then `navigate`. But it was not broken by this PR